src/icons/general/IconEyeHidden.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconEyeHidden = ({ className = '', color = '#222', height = 16, title, width = 16 }: Props) => (
+    <AccessibleSVG
+        className={`bdl-IconEyeHidden ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 16 16"
+        width={width}
+    >
+        <g className="fill-color" fill={color} fillRule="evenodd">
```

Comment:
I think you can move `className="fill-color" fill={color}` to the paths below and get rid of the g and one nesting level